src/components/WalletModal/index.tsx

```diff
@@ -15,7 +15,7 @@ import usePrevious from '../../hooks/usePrevious'
 import { ApplicationModal } from '../../state/application/actions'
 import { useModalOpen, useWalletModalToggle } from '../../state/application/hooks'
 import { ExternalLink, TYPE } from 'theme'
-import AccountDetails from 'components/AccountDetails'
+import AccountDetails from '@src/components/AccountDetails'
```

Comment:
Thanks @anxolin now seeing you do it, makes sense. As the original UNI file was pulling our mod file.